shell.js

```diff
@@ -72,15 +87,15 @@ exports.tail = common.wrap('tail', _tail, {idx: 1});
 //@include ./src/toEnd
 
 //@include ./src/sed
-var _sed = require('./src/sed');
+var _sed = require('./src/sed'); // don't glob-expand regexes
```

Comment:
Good point. I think I put it there just because the line was getting long, but I agree it makes more sense as you suggested.